app/assets/stylesheets/components/_tooltip.scss

```diff
@@ -12,6 +12,107 @@
     white-space: normal;
     width: 100%;
   }
+
+  &:focus::before,
+  &:focus::after {
+    opacity: 1;
+    transition-delay: 100ms;
+    visibility: visible;
+  }
+}
+
+.hint--top {
+  &:focus::before {
+    transform: translateY(-8px);
```

Comment:
can do you feel about this `$space-1` instead of `8px` throughout this file? also, are we ever gonna have a tooltip that isn't on the top of the element? to me, to feel likes no; if you agree, maybe we should remove the left, right, bottom stuff??